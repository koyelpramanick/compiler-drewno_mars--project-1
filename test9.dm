"\\""
BOOTY!
"you can escape a /b and leave the quote open!
silly too hot goose
"making sure 24Kmagic isn't recognized inside string"
fart != toot
return <=, >=, :
<; >; ? //wowwowowo
7--
