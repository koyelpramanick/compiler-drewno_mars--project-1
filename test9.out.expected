STRINGLITERAL:"\\" [1,1]
ID:BOOTY [2,1]
NOT [2,6]
ID:silly [4,1]
FALSE [4,7]
ID:goose [4,15]
STRINGLITERAL:"making sure 24Kmagic isn't recognized inside string" [5,1]
ID:fart [6,1]
NOTEQUALS [6,6]
ID:toot [6,9]
RETURN [7,1]
LESSEQ [7,8]
COMMA [7,10]
GREATEREQ [7,12]
COMMA [7,14]
COLON [7,16]
LESS [8,1]
SEMICOL [8,2]
GREATER [8,4]
SEMICOL [8,5]
INTLITERAL:7 [9,1]
POSTDEC [9,2]
EOF [10,1]
